packages/documentation/src/pages/platform/front-end-standards/security

```diff
@@ -0,0 +1,27 @@
+# Front end security 
+
+`vets-website` uses modern browser features to provide a secure experience for veterans. In general, these features are automatically included through the deployment process and require no configuration by front end engineers. 
+
+_This is an overview not an implementation guide. These rules apply to VA.gov production environments. Not all security features will be used in local, development, or staging environments._ 
+
+## HTTPS
+
+- All connections made by VA.gov must be secure (HTTPS)
+- VA.gov automatically redirects to a secure protocol if a non secure connection is attempted 
+
+## Cross-Origin Resource Sharing (CORS)
+
+- Some cross origin connections made by VA.gov must support CORS including **all** XHR connections and web font downloads. 
+- Updates to CORS headers returned by `vets-api` services or `vets-website` assets must be made through the devops team and must be reviewed by the [frontend review group](https://github.com/orgs/department-of-veterans-affairs/teams/frontend-review-group). Example:  
+  - Your application is on a VA.gov subdomain, needs to connect to `vets-api`, but is not currently listed in the allowed origins
+- More [info on cors](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS)
+
+## Content Security Policy (CSP)
+
+- All VA.gov pages must return a Content Security Policy header that allows only known scripts to run on the page
+- The CSP must be *enforced* i.e. not set to `report-only`
+- The `report-uri` should point to a VA platform error capturing service (Sentry)
+  - VA.gov throttles these reports by including the `report-uri` only on a small % of responses
+- Updates to CSP headers returned by or `vets-website` assets must be made through the devops team and must be reviewed by the [frontend review group](https://github.com/orgs/department-of-veterans-affairs/teams/frontend-review-group)
+- More [info on CSP](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP)
+
```

Comment:
should we include something about how staging.va.gov is gated? e.g. all test environment should at least require simple password access (dunno what terminology to use here).

Reply:
It's not gated right now, yeah? Once upon a time, we had HTTP basic auth, but that was removed a while ago.